profile/README.md

```diff
@@ -0,0 +1,85 @@
+schedule, API design, and coding style. Our current focus is on stabilising these crates.
+
+Name | Crate | Repository
+---|---|---
+base58 | [base58ck](https://crates.io/crates/base58ck) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoin | [bitcoin](https://crates.io/crates/bitcoin) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoinconsensus | [bitcoinconsensus](https://crates.io/crates/bitcoinconsensus) | [rust-bitcoinconsensus](https://github.com/rust-bitcoin/rust-bitcoinconsensus)
+bech32 | [bech32](https://crates.io/crates/bech32) | [rust-bech32](https://github.com/rust-bitcoin/rust-bech32)
+hashes | [bitcoin_hashes](https://crates.io/crates/bitcoin_hashes) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+hex | [hex-conservative](https://crates.io/crates/hex-conservative) | [hex-conservative](https://github.com/rust-bitcoin/hex-conservative)
+internals | [bitcoin-internals](https://crates.io/crates/bitcoin-internals) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+io | [bitcoin-io](https://crates.io/crates/bitcoin-io) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+ordered | [ordered](https://crates.io/crates/ordered) | [rust-ordered](https://github.com/rust-bitcoin/rust-ordered)
+secp256k1 | [secp256k1](https://crates.io/crates/secp256k1) | [rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1)
+units | [bitcoin-units](https://crates.io/crates/bitcoin-units) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+
+We aim to make usage of these crates secure, ergonomic, and intuitive. We try to enable users of
+these crates to follow software engineering best practices (eg, semver compliance).
+
+If we fall short of these stated aims, or can in anyway improve our support of Bitcoin free software
+projects, please raise issues or contact maintainers directly.
+
+If you write proprietary software and want better integration with these crates by all means raise
+issues on the respective repository. We are interested in all users and you usecase may help us
+improve the libraries, however all development is done in the open and will stay that way.
+
+## Other Crates
+
+Other crates/repositories hosted here and developed and maintained by the community.
+
+These crates are developed and maintained by a subset of the rust-bitcoin devs or other members of
+the community. We are quite open with hosting crates in the org, as such the bar for entry is only:
+
+- Repository contains Bitcoin specific crates (implies Bitcoin Only and not general purpose crates)
+- Repository has at least minimal CI
+- Author has made at least some effort to adhere to practices and standards maintained in the core crates.
+
+As such any of these crates should be personally reviewed by projects before use, including
+reviewing new versions and not just blindly upgrading - you have been warned.
```

Comment:
```
These crates may not be reviewed or maintained by the primary rust-bitcoin maintainers. Keep this in mind when choosing to use them as dependencies. And as with any dependencies, you should review new versions rather than blindly upgrading.
```

Reply:
We don't want to imply that these crates are *bad* (otherwise we really would move them out of the org). Just that they have a different trust profile than the main crates.